### PR TITLE
Improve description of the current CI situation

### PR DESCRIPTION
**Pull request by jbergstroem**
A bit short on text, but at least correctly describe the current situation. Not sure we should expand this too much about what we want in our future CI software; that probably should live in its own place.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,11 +81,9 @@ The following platforms will eventually be included in the build and **test-all*
 CI Software
 -----------
 
-Build and test orchestration will be primarily performed by [Buildbot](http://buildbot.net/). Each build platform will run a Buildbot slave and they will communicate with a Buildbot master that will coordinate both io.js and libuv.
+Build and test orchestration is performed by [Jenkins](http://jenkins-ci.org). You can find a summary of build status [here](https://jenkins-iojs.nodesource.com).
 
-A small io.js application will watch the io.js and libuv projects on GitHub and request builds from the Buildbot master and provide feedback to GitHub. The scope of the io.js application may increase over time, potentially leaving Buildbot to perform orchestration only.
-
-The goal is to invest in io.js-specific CI infrastructure and tooling while leaning on existing, proven technologies where appropriate.
+Our ambition is to invest in io.js-specific CI infrastructure and tooling while leaning on existing, proven technologies where appropriate. This will slowly replace Jenkins as orchestrator and buildslaves throughout our CI ecosystem.
```
Comment (kenperkins) [inline README.md]: This is totally a nit, but maybe be a bit more aspirational here? `We hope to slowly replace...` ?

Comment (jbergstroem) [inline README.md]: Updated, thanks.

**Comment by wblankenship** [comment]
:heart:

**Comment by kenperkins** [comment]
lgtm :+1:

**Comment by jbergstroem** [comment]
Landed in 6a10343800bf47202d545fe492e91b3066138720. Thanks.

**Comment by wblankenship** [comment]
I'm new to this workflow on GitHub. Instead of directly merging the pull request, what did you do to produce https://github.com/iojs/build/commit/6a10343800bf47202d545fe492e91b3066138720?

**Comment by jbergstroem** [comment]
@wblankenship I wasn't entirely sure what was expected either, but the general pattern I've noticed on `io.js/iojs` is that you either merge someone else's PR (for instance https://github.com/iojs/io.js/commit/97b424365a883f3b8de18b3ec3f256307a92ad09) or just fast forward and merge your own pr branch with reviewers in commit message (https://github.com/iojs/io.js/commit/b9686233fc0be679d7ba1262b611711629ee334e). Since we haven't landed our own guidelines I followed their (which ours in progress are pretty similar to if I remember correctly).

**Pull request closed without merging**